_examples/amqpstreaming.adoc

```diff
@@ -1,4 +1,5 @@
 = AMQP - Spark Streaming
+:page-project-name: Examples
```

Comment:
technically not really, i just gave them all the same project name for convenience and because i wasn't quite sure what to do.

so, that `page-project-name` header was created to address the mapping between old project names (eg Ophicleide) and the newly proposed project titles. basically, when i started to convert the titles, the menu bar entries looked crazy with too many words. it was really inconsistent and didn't fit the other menu entries. so, i put the long title as the header on each page, which i think makes it really easy to tell where you are, and then put the old project names on the menu bar since we reference those names in the docs.

the example name titles are kinda all over the place and i didn't want to put them in the header. also, since i would like to migrate the examples to "how do i?" articles in the near future i figured it was be easy to just generically mark them as "Examples".

all that said, i'm open to suggestions =)

Reply:
I see. No, that seems fine.